### ISSUE TITLE
Zrobić jakieś serializatory/deserializatory w konfigach.

### ISSUE THREAD

**Issue by Rollczi** [FunnyGuilds/FunnyGuilds#1673]
Zrobić jakieś serializatory/deserializatory w konfigach dla np. takiego Material.
https://github.com/FunnyGuilds/FunnyGuilds/pull/1671

**Comment by Rollczi** [comment]
Wydaje mi się że jeszcze będzie to kontynuowane.

**Comment by P3ridot** [comment]
Tak, ja dodałem tylko kilka najprostszych transformerów a jakieś by sie pewnie jeszcze znalazły np. do ItemStacków
